extensions/amp-list/0.1/amp-list.css

```diff
@@ -46,6 +46,11 @@ amp-list[load-more] [load-more-button].i-amphtml-default-ui,
   text-align: center;
 }
 
+[load-more-button].i-amphtml-list-load-more-overflow.i-amphtml-default-ui {
```

Comment:
this should not be specific to our `.i-amphtml-default-ui` class and should apply to all cases.

Reply:
Done.